libraries/Microsoft.Bot.Builder.Expressions/SimpleObjectScope.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Expressions
+{
+    internal class SimpleObjectScope : IMemoryScope
+    {
+        private object scope = null;
```

Comment:
Please make sure you clean up all the visual studio (blue) warnings about style.  I suspect you are introducing some...